### ISSUE TITLE
False Positive "no member" with numpy finfo

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#10838]
### Bug description

```python
"https://github.com/pylint-dev/pylint/issues/10838"
import numpy as np
print(np.finfo(1.0).eps)
```

### Command used

```shell
pylint pylint-10838.py
```

### Pylint output

```python
pylint-10838.py:3:6: E1101: Instance of 'finfo' has no 'eps' member (no-member)
```

### Expected behavior

no diagnostics.
the code runs as expected, printing `2.220446049250313e-16`

### Pylint version

```shell
pylint 4.0.4
astroid 4.0.3
Python 3.14.2 (tags/v3.14.2:df79316, Dec  5 2025, 17:18:21) [MSC v.1944 64 bit (AMD64)]
```

### OS / Environment

windows

### Additional dependencies

```python
numpy==2.4.1
```

**Comment by wshanks** [comment]
Previously reported in https://github.com/pylint-dev/pylint/issues/10806.
